build.gradle

```diff
@@ -308,6 +308,13 @@ def distributionCodeGen = copySpec {
   include '**/*'
 }
 
+def distributionRarInfo = copySpec {
+  from rootProject.file('rarinfo/src/dist/resources/')
+  into 'doc/rarinfo'
+  include '**/*'
+}
```

Comment:
well I've forgotten to mention, but you need an update to rar-info.sh and rar-info.bat since in our new distribution ironjacamar-rarinfo.jar is not local to doc/rarinfo, but it lives in lib as any other jar